platform_versioned_docs/version-23.3.0/enterprise/kubernetes.mdx

```diff
@@ -68,82 +71,82 @@ Seqera Platform Enterprise is distributed as a collection of Docker containers a
     ```
 
 :::note
-The `configmap.yml` manifest includes both the `tower.env` file and `tower.yml` file. These files are made available to the other containers through volume mounts.
+The `configmap.yml` manifest includes both the `tower.env` and `tower.yml` files. These files are made available to the other containers through volume mounts.
 :::
 
 ### Redis
 
 Seqera Enterprise requires a Redis database for caching purposes. Configure Redis manually by deploying a manifest to your cluster, or configure a managed Redis service.
 
-**Deploy a Redis manifest to your cluster**
+#### Deploy a Redis manifest to your cluster
 
 1. Download the appropriate manifest for your infrastructure:
 
     - [Azure AKS](_templates/k8s/redis.aks.yml)
-
     - [Amazon EKS](_templates/k8s/redis.eks.yml)
-
     - [Google Kubernetes Engine](_templates/k8s/redis.gke.yml)
 
-2. Deploy to your cluster:
+1. Deploy to your cluster:
```

Comment:
I wonder if this point and the preceding one can be sentences rather than numbered items - they're consecutive steps but don't need numbering.

Reply:
Feels awkward to combine the consecutive steps into one sentence/ paragraph. I'd rather keep them as is :)